start-a-local-cluster.md

```diff
@@ -31,37 +31,35 @@ Data is stored on-disk and client/server communication is secure. Setup involves
     # To exit: CTRL + D.
     ```
 
-4. [Run some queries](/basic-sql-commands.html).
+3. [Run some queries](/basic-sql-commands.html).
+4. Check out the Admin UI by pointing your browser to `http://<your local host>:26257`. You can find your local host by running `hostname` in your shell.    
 
 ## Standard Mode
 
 1. From the directory containing the `cockroach` binary, initialize the cluster:
 
-    ```bash
-    $ ./cockroach init --stores=ssd=data
     ```
-    The `--stores` flag sets the path for local storage.
+    $ ./cockroach init --stores=ssd=temp/data
+    ```
+    where `ssd` can be any arbitrary string describing the store (e.g., `ssd` for flash, `hdd` for spinny disk) and `temp/data` is the filepath to the storage location. For the filepath, the parent directory must exist and the store directory, if it already exists, must not contain any CockroachDB data.
 
-2. Create certificates for the database:
+2. Create security certificates:
 
     ```bash
     $ ./cockroach cert create-ca
     $ ./cockroach cert create-node localhost $(hostname) 
+    $ ./cockroach cert create-client root
     ```
+    These commands create security certificates in the `certs` directory. The first two commands create the files for the cluster: `ca.cert`, `ca.key`, `node.server.crt`, and `node.server.key`. The last command creates the files for the SQL client: `root.client.crt`, `root.client.key`, `node.client.crt`, and `node.client.key`.  
```

Comment:
`node.client.crt` and `node.client.key` are created by the second command (`cockroach cert create-node`)